lib/synapse/haproxy.rb

```diff
@@ -734,13 +740,28 @@ def generate_backend_stanza(watcher, config)
       else
         backends.keys.shuffle
       end
+
+      cookie_value_method = "name"
```

Comment:
This seems redundant to the block down below. Let's just do the case down below (I'll comment with a suggestion)